go.mod

```diff
@@ -1,6 +1,9 @@
 module gobot.io/x/gobot/v2
 
-go 1.17
+go 1.18
+
+// this causes problems during tests
+exclude github.com/nats-io/nats.go v1.26.0
```

Comment:
Maybe this can be fixed with linter finding: SA1019: o.TLSConfig.RootCAs.Subjects has been deprecated since Go 1.18: if s was returned by SystemCertPool, Subjects will not include the system roots

Reply:
fixed